roles/k3s_setup/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Add IP address of all hosts to all hosts
+  lineinfile:
+    dest: /etc/hosts
+    regexp: '.*{{ item }}$'
+    line: "{{ hostvars[item].ansible_host }} {{item}}"
+    state: present
+  when: hostvars[item].ansible_host is defined
+  with_items: "{{ groups.all }}"
```

Comment:
Use `loop` instead of `with_X`. `with_X` will be deprecated soon.

https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html#comparing-loop-and-with.

Reply:
Updated.